src/diffusers/pipelines/stable_diffusion/pipeline_stable_diffusion.py

```diff
@@ -264,152 +264,64 @@ def _encode_prompt(
         r"""
         Encodes the prompt into text encoder hidden states.
 
-        Args:
-             prompt (`str` or `List[str]`, *optional*):
-                prompt to be encoded
-            device: (`torch.device`):
-                torch device
-            num_images_per_prompt (`int`):
-                number of images that should be generated per prompt
-            do_classifier_free_guidance (`bool`):
-                whether to use classifier free guidance or not
-            negative_prompt (`str` or `List[str]`, *optional*):
-                The prompt or prompts not to guide the image generation. If not defined, one has to pass
-                `negative_prompt_embeds` instead. Ignored when not using guidance (i.e., ignored if `guidance_scale` is
-                less than `1`).
-            prompt_embeds (`torch.FloatTensor`, *optional*):
-                Pre-generated text embeddings. Can be used to easily tweak text inputs, *e.g.* prompt weighting. If not
-                provided, text embeddings will be generated from `prompt` input argument.
-            negative_prompt_embeds (`torch.FloatTensor`, *optional*):
-                Pre-generated negative text embeddings. Can be used to easily tweak text inputs, *e.g.* prompt
-                weighting. If not provided, negative_prompt_embeds will be generated from `negative_prompt` input
-                argument.
-            lora_scale (`float`, *optional*):
-                A lora scale that will be applied to all LoRA layers of the text encoder if LoRA layers are loaded.
+        The call is forwarded to [`StableDiffusion.encode_prompt_from_cls`].
```

Comment:
>  also don't have it be displayed in the docs.

How do we ensure it? Instead of doing `all` under a class header in the docs, we just manually choose the method names we would like to display from the docs?

Reply:
This could be better handled with some decorator no? Like If I decorate a method with something like `@no_render` the `docs-builder` should be able to pick that up and not display it on the underlying doc page. To me this seems like a better solution. WDYT?